uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_absence-tab-page.scss

```diff
@@ -0,0 +1,6 @@
+.absence-tab-page {
+
+  .btn-group {
+    margin-bottom: 20px !important;
```

Comment:
earlier the style was not working so made it important to see and forgot to remove it